drivers/hmis_csv_importer/app/jobs/hmis_csv_importer/cleanup/expire_base_job.rb

```diff
@@ -52,13 +52,8 @@ def perform(model_name: nil, retain_item_count: 5, retain_after_date: DateTime.c
         model_name: next_model.name,
         retain_item_count: retain_item_count,
         retain_after_date: retain_after_date,
-        delay: delay,
       }
-      if delay
-        self.class.perform_later(**options)
-      else
-        self.class.perform_now(**options)
-      end
+      self.class.perform_later(**options)
```

Comment:
This needs to be `perform_now` or else it will end up being run as a DJ.   (Note the default was `delay: false`.)  I actually think we should keep this around until we're happy with the behavior in production.

Reply:
The idea is that this _should_ be called in sequence within the one cron task.  Cron tasks can run indefinitely (I believe) but delayed jobs have a max runtime.

Reply:
That's fine but I think having the job call itself recursively for each model like this is not ideal. I'll adjust this to run over all models discretely in series